packages/js/create-product-editor-block/plugin-templates/$slug.php.mustache

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Plugin Name:       {{title}}
+{{#pluginURI}}
+ * Plugin URI:        {{{pluginURI}}}
+{{/pluginURI}}
+{{#description}}
+ * Description:       {{description}}
+{{/description}}
+ * Version:           {{version}}
+ * Requires at least: 6.1
+ * Requires PHP:      7.0
```

Comment:
Maybe update this to `7.3` to match WooCommerce.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/38263/commits/5965be42582ea3ab2089d822fb2f1ea716da0ef7

I also added a minimum WC version in https://github.com/woocommerce/woocommerce/pull/38263/commits/4e8651e8bf2ef56aee26c727cc974d5da5e8f8b1